common.blocks/progressbar/progressbar.js

```diff
@@ -24,7 +24,9 @@ provide(BEMDOM.decl(this.name, /** @lends progressbar.prototype */{
      * @returns {progressbar} this
      */
     setVal : function(val) {
-        this.elem('bar').css('width', (this._val = val) + '%');
+        this.elem('bar')
+            .css('width', (this._val = val) + '%')
+            .attr('aria-valuenow', this._val);
```

Comment:
But specification says that value should be as a number. I don't add `aria-value(min/max)` so `aria-valuenow` will be as `%` by default.